registry.json

```diff
@@ -4,10 +4,15 @@
      "updated": "2021-06-24T01:06:23+0000"
   },
   "registry": {
+     "did:web:digitalcredentials.github.io": {
```

Comment:
Is this the correct location for MIT's DID document?

Reply:
@stuartf @kezike - no, that's not the correct location for the moment (there is no DID Document hosted at that url).

Reply:
@stuartf @kezike - I added an Example University `did:key` entry, which we'll be using on the playground, while DCC is still setting up its DID Doc.